simple/components/webui/scripts/create.sh

```diff
@@ -46,4 +46,22 @@ ctx logger info "Deploying Grafana Configuration..."
 grafana_conf=$(ctx download-resource "components/webui/config/config.js")
 sudo mv ${grafana_conf} "${GRAFANA_HOME}/config.js"
 
+ctx logger info "Configuring logrotate..."
+lconf="/etc/logrotate.d/nodejs"
+
+cat << EOF | sudo tee $lconf > /dev/null
+$WEBUI_LOG_PATH/*.log {
```

Comment:
I actually think that I saw that the log4js config in gsPresets.json already rotates logs. Are there any additional logs to rotate?